### PR TITLE
Changelog 8.4

### PR DESCRIPTION
**Pull request by lehins**
# Description

List of high level things that made it into 8.4 and initial list of such things that will make it into next 8.5 release

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/input-output-hk/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [ ] Self-reviewed the diff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
...
 [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
+## 8.5
+
+- Prevent `DRep` expiry when there are no Governance Actions.
+- Refreshing of expiry with votes and `UpdateTxCert`
+- Disable ability to submit transactions with votes by Stake Pool Operators and
+  Constitutional Committee members on Governance Actions that they should not
+  be able to vote on.
+- Add ability to supply initial Constitutional Committee and initial version of Contitution.
```
Comment (teodanciu) [inline CHANGELOG.md]: `Contitution` -> `Constitution`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
...
+- Preserve order of submitted ProposalProcedures and account for their priority.
+- Disallow empty fields in CBOR of Conway TxBody
+- Implement Constitutional Committee expiration, validation and modification
+
```
Comment (teodanciu) [inline CHANGELOG.md:21]: How about [this](https://github.com/input-output-hk/cardano-ledger/commit/16a9090d62bacaf441a4da9726cf2483d006e0ee) commit ?  `Add conway-specific certs to deposit/refunds`

But I don't understand it very well, maybe it's not something that is reflectable in changelog..., please ignore if so

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,29 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
 If you are looking for the Ledger Releasing and Versioning Process then you can find it in
 [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
+## 8.5
```
Comment (teodanciu) [inline CHANGELOG.md:10]: I'm unclear with the versions, 
I guess 8.4 is cardano-ledger-conway-1.8.0.0  and 
8.5 will be whatever we'll release this week?

**Review by teodanciu — APPROVED** [review]
Looks good to me

File: CHANGELOG.md (hunk 3 above)
Comment (lehins) [reply CHANGELOG.md:10]: Yep. That is correct

File: CHANGELOG.md (hunk 1 above)
Comment (lehins) [reply CHANGELOG.md]: Done. Thanks.

**Pull request merged**